log/mod.ts

```diff
@@ -418,183 +387,10 @@ const state = {
  * and {@linkcode TestHandler}.
  */
 
-export const formatters: {
-  jsonFormatter(logRecord: LogRecord): string;
-} = {
-  jsonFormatter,
-};
-
-/** Get a logger instance. If not specified `name`, get the default logger. */
-export function getLogger(name?: string): Logger {
-  if (!name) {
-    const d = state.loggers.get("default");
-    assert(
-      d !== undefined,
-      `"default" logger must be set for getting logger without name`,
-    );
-    return d;
-  }
-  const result = state.loggers.get(name);
-  if (!result) {
-    const logger = new Logger(name, "NOTSET", { handlers: [] });
-    state.loggers.set(name, logger);
-    return logger;
-  }
-  return result;
-}
-
-/** Log with debug level, using default logger. */
-export function debug<T>(msg: () => T, ...args: unknown[]): T | undefined;
-export function debug<T>(
-  msg: T extends GenericFunction ? never : T,
-  ...args: unknown[]
-): T;
-export function debug<T>(
-  msg: (T extends GenericFunction ? never : T) | (() => T),
-  ...args: unknown[]
-): T | undefined {
-  // Assist TS compiler with pass-through generic type
-  if (msg instanceof Function) {
-    return getLogger("default").debug(msg, ...args);
-  }
-  return getLogger("default").debug(msg, ...args);
-}
-
-/** Log with info level, using default logger. */
-export function info<T>(msg: () => T, ...args: unknown[]): T | undefined;
-export function info<T>(
-  msg: T extends GenericFunction ? never : T,
-  ...args: unknown[]
-): T;
-export function info<T>(
-  msg: (T extends GenericFunction ? never : T) | (() => T),
-  ...args: unknown[]
-): T | undefined {
-  // Assist TS compiler with pass-through generic type
-  if (msg instanceof Function) {
-    return getLogger("default").info(msg, ...args);
-  }
-  return getLogger("default").info(msg, ...args);
-}
-
-/**
- * @deprecated (will be removed after 0.214.0) Use {@linkcode warn} instead.
- *
- * Log with warning level, using default logger.
- */
-export function warning<T>(msg: () => T, ...args: unknown[]): T | undefined;
-/**
- * @deprecated (will be removed after 0.214.0) Use {@linkcode warn} instead.
- *
- * Log with warning level, using default logger.
- */
-export function warning<T>(
-  msg: T extends GenericFunction ? never : T,
-  ...args: unknown[]
-): T;
-export function warning<T>(
-  msg: (T extends GenericFunction ? never : T) | (() => T),
-  ...args: unknown[]
-): T | undefined {
-  // Assist TS compiler with pass-through generic type
-  if (msg instanceof Function) {
-    return warn(msg, ...args);
-  }
-  return warn(msg, ...args);
-}
```

Comment:
Where did these exports go? They should probably live in `log/warn.ts` until they're deprecated.

Reply:
I thought it can be removed together. 

Updated.